frontend/packages/dev-console/src/components/svg/SvgBoxedText.tsx

```diff
@@ -123,7 +124,7 @@ const SvgBoxedText: React.FC<SvgBoxedTextProps> = ({
             y={y + iconPadding}
             width={30}
             height={30}
-            xlinkHref={getImageForIconClass(`icon-${icon}`)}
+            xlinkHref={isIconUrl(icon) ? icon : getImageForIconClass(`icon-${icon}`)}
```

Comment:
@jeff-phillips-18 FYI, I have this use case where i have `data:url` from the csv instead of icon name. I believe you have the proper icon alignment in one of your PRs.

Reply:
thanks, I'll be sure to adjust.